pkg/collector/collector.go

```diff
@@ -86,7 +86,13 @@ func main() {
 	}
 
 	if *scanDisabled {
-		file, err := os.OpenFile("/run/eraser.sh/shared-data/scanErase", os.O_WRONLY, os.ModeNamedPipe)
+		_, err := os.Create("/run/eraser.sh/shared-data/scanErase")
```

Comment:
Should replace this with https://pkg.go.dev/golang.org/x/sys/unix#Mkfifo
Mode would likely need to be 0644 if the read side is running under a different UID.